### ISSUE TITLE
Allow UdonSharp to be downloaded via unitypackage

### ISSUE THREAD

**Issue by dark-swordsman** [vrchat-community/UdonSharp#95]
**Feature Description:**
Allow UdonSharp to be downloaded via unitypackage

**Additional context:**
Many users still have not downloaded the VCC. There is still the ability for people to create projects through the template repos on github. However, there doesn't seem to be a way to install the latest version of UdonSharp, which a lot of projects rely on.

**Comment by Pentakill1995** [comment]
YES! Fuck this companion shit. i hate it. why is vrchat such a shit base for creators

**Comment by Happyrobot33** [comment]
Annoyingly if the releases were automated here it would be possible to install without VCC since all the VCC is doing is installing unity packages behind the scenes. Honestly though there really isnt much of a point *not* using the VCC at this point since its basically just a package manager and thats it
